### PR TITLE
kubemacpool: Use podman on presubmit jobs

### PR DESCRIPTION
**Pull request by oshoval**
Signed-off-by: Or Shoval <oshoval@redhat.com>

### CODE REVIEW

File: github/ci/prow-deploy/files/jobs/k8snetworkplumbingwg/kubemacpool/kubemacpool-presubmits.yaml [hunk 1]
```diff
@@ -15,19 +15,32 @@ presubmits:
...
       max_concurrency: 6
       labels:
-        preset-dind-enabled: "true"
+        preset-podman-in-container-enabled: "true"
         preset-docker-mirror-proxy: "true"
         preset-shared-images: "true"
       spec:
         containers:
-          - image: quay.io/kubevirtci/kubevirt-infra-bootstrap:v20220425-b167bb8
+          - image: quay.io/kubevirtci/bootstrap:v20220816-91d8af2
```
Comment (oshoval) [inline github/ci/prow-deploy/files/jobs/k8snetworkplumbingwg/kubemacpool/kubemacpool-presubmits.yaml:23]: @brianmcarey 
I remember you updated `quay.io/kubevirtci/kubevirt-infra-bootstrap` to  `quay.io/kubevirtci/bootstrap` right ?
here we still see the old ones before this PR changes, did you revert it ? or it is an additional place?

Comment (brianmcarey) [reply github/ci/prow-deploy/files/jobs/k8snetworkplumbingwg/kubemacpool/kubemacpool-presubmits.yaml:23]: it hasn't been merged yet but the rehearsals seemed to work fine with the regular bootstrap image - https://github.com/kubevirt/project-infra/pull/2226

Comment (oshoval) [reply github/ci/prow-deploy/files/jobs/k8snetworkplumbingwg/kubemacpool/kubemacpool-presubmits.yaml:23]: thanks, maybe lets merge it ?

**Comment by oshoval** [comment]
/cc @brianmcarey 
We got a problem here 
https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_project-infra/2128/rehearsal-pull-kubemacpool-e2e-k8s/1561997704249217024

It does works btw on other lanes with same image
for example
https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_project-infra/2125/rehearsal-pull-e2e-cluster-network-addons-operator-br-marker-functests/1561992160264327168

Maybe need to extend timeout ?
maybe we have pid leakage ? (saw some other commands, not related to podman that fails here and there,
iptables commands IIRC)

**Comment by oshoval** [comment]
https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_project-infra/2128/rehearsal-pull-kubemacpool-e2e-k8s/1562010225144238080

passed after rerunning
I believe it is something like max files opened at the same time on CI or so

**Comment by brianmcarey** [comment]
> /cc @brianmcarey We got a problem here https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_project-infra/2128/rehearsal-pull-kubemacpool-e2e-k8s/1561997704249217024

Thats the first time I've seen it hit the timeout. Of course if we start to see it happen more regularly than I can increase the timeout but it normally starts almost immediately.

**Comment by oshoval** [comment]
> > /cc @brianmcarey We got a problem here https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_project-infra/2128/rehearsal-pull-kubemacpool-e2e-k8s/1561997704249217024
> 
> Thats the first time I've seen it hit the timeout. Of course if we start to see it happen more regularly than I can increase the timeout but it normally starts almost immediately.

It waited i think 15 seconds (1+2...+5), so maybe timeout won't help tbh
maybe something like max_files open and then the service command itself fails ?
But np, we can check next time it happens
thanks

**Review by RamLavi — APPROVED** [review]
/lgtm

**Comment by oshoval** [comment]
Thanks

@qinqon @phoracek  
can you take a look please ?

**Review by phoracek — COMMENTED** [review]
/approve

**Pull request merged**